### PR TITLE
Move DannyLuu:dev (#577) to evancohen:dl/fitbitSquash

### LINKED ISSUE #519: Light settings : Hue type in selector is not saved
On the remoteUI, in the Light Settings the "Hue" type in the selector is not saved after submission

### Expected Behavior
 

Change the select from "hyperion" to "hue" save and use the miroir to light up the room 

### Current Behavior

On the remoteUI, in the Light Settings I set the Hue ip and hue username  
in the setup part I add a name "Room", a target "Item1"  select the type "Hue" (and not "hyperion") and the id to "1"  
I save
I submit the changes the type is set to "hyperion" (and in the config file too)
the light doesn't work

### Possible Solution

Save the "hue" type in the file
(I actually did that manually)

### Steps to Reproduce (for bugs)

1. go to the remoteUI params
2. add a light with "hue" type
3. save
4. reopen the light settings

### Your Environment

* Environment name and version (e.g. Raspberry Pi 3, node.js 5.4):
Rapsberry 3, ps EYE, node v6.9.2 , npm v3.10.9
* Operating System and version (e.g. Raspbian, Ubuntu, OSX):
Linux raspberrypi 4.4.34-v7+ #930 SMP Wed Nov 23 15:20:41 GMT 2016 armv7l GNU/Linux

### PR DESCRIPTION
**Pull request by justbill2020**
* removed "npm" and "install" packages from dependencies

* fix for conf-audio.sh

removes the comment line from exported file contents

* Update config.schema.json

fixes #519 @dannyluu i just updated the part that allows it to save... the UI still needs to be fixed... please keep working on it...

* Add missing brace. (#542)

* Adding eslint + travis (#548)

* Adding eslint + travis

* Remove Dependency CI and show build status

* Fixing path problems and first run (#549)

* use path.resolve() to get files

fixing up sonus.js

* updating speech config schema

* remove umdl beacuse we aren't using it anymore

* fix sonus error on first run

* better error message on first run

* oh no. :cold_sweat:

* don't use kiosk mode when started with dev :rocket:

* Remove error when Stock names are not filled. (#551)

* Remove error when Stock names are not filled.

* Add missing semi-colon

* Set refresh to 30mins.

* Fix browser console errors with null tv show lists. (#552)

* Fix browser console errors with null tv show lists.

* Fix build error. I am new to angular?!

* Angular doesn't need double negations?

* Fixed error with language parameter and weather (#550)

* Fixed typo in language variable name

* Fixed typo in language variable name

* Fixed wrong moment.js lang parameter

* Changed handling of the language variable handling for moment.js

* v0.0.10

* Create alias for starting the smart mirror from anywhere. (#555)

* Create alias for starting the smart mirror from anywhere.

* Update smart mirror!

* Update bashrc with subshell function.

* Reload bash profile upon install.

* Update install message to reflect new changes

* fixing fitbit

* Add english resource for Fitbit.

* Cycle through Fitbit statistics.

* Update controller documentation and optimized service calls.

* Update fitibt css for progress bar.

* Add floors to Today's stats and fixed other tags.

* Update fitbit EN resource.

* Shortened the progress bar.

* Update init logic... my bad.

* Show progress bar on daily stats.

* Update progress bar ot turn green when 100%

* Localizing units

* Update ui to display green progress bar when over 100%

* Adding comments for new enpoints that I would like to integrate.

* Update english locale.

* Update service and controller with new api requests.

* Update fitbit index html to incorporate controller function calls.

* Adding all changes.

* Setting groups to cycle.

* Finalizating changes.

* Updated views on data displayed.

* Add message resources

Resources added for the languanges. WIll need translations

* Replaced spaces with tabs

* Found the culprit!

####  Fixes Related issues
- add related
- issues here

####  Checklist
- [ ] I have read and understand the CONTRIBUTIONS.md file
- [ ] I have searched for and linked related issues
- [ ] I have added config.schema.json file if config option are required.
- [ ] I am NOT targeting master branch

### CODE REVIEW

**Comment by evancohen** [comment]
Ooh, ok. That's how you squashed it. Perf :)

My earlier comment still stands: can we just use google translate to localize those strings?

**Comment by justbill2020** [comment]
why are we not using google translate for all translation for every language that is available?

**Comment by evancohen** [comment]
I don't think that would work for voice commands, grammatically things will be too different. For static strings like these I think it would be a good idea.

File: scripts/pi-install.sh [hunk 1]
```diff
@@ -160,11 +160,11 @@ cat << "EOF"
...
         |        The smart-mirror is now installed!
        / \       
-      / _ \      To configure audio input, speech, and the mirror itself, please visit:
+      / _ \      For configuration instructions, check out the documentation at:
      |.o '.|     http://docs.smart-mirror.io
      |'._.'|     
-     |     |     To start your mirror:
-   ,'|  |  |`.   > mirror start
+     |     |     To start your mirror you can run both on the pi or via SSH:
+   ,'|  |  |`.   > start-mirror
```
Comment (evancohen) [inline scripts/pi-install.sh]: You should revert your changes to this file, this will break the new way we want to run the mirror.

Comment (DannyLuu) [inline scripts/pi-install.sh]: Frig! How did this shit get there...

**Comment by DannyLuu** [comment]
I noticed that when we save config changes, my fitbit info gets removed. Along with that when there is no fitbit info, the display still cycles. We need to disable it if fitbit is not setup.

**Comment by DannyLuu** [comment]
Alright after reviewing the changes, the following needs to be done in order for a successful merge:

- [ ] Create remote UI to save fitbit credentials

- [ ] Have a button to authenticate the fitbit token

**Pull request merged**